packages/ses/src/commons.js

```diff
@@ -174,6 +176,19 @@ export const arraySome = uncurryThis(arrayPrototype.some);
 export const arraySort = uncurryThis(arrayPrototype.sort);
 export const iterateArray = uncurryThis(arrayPrototype[iteratorSymbol]);
 //
+export const arrayBufferSlice = uncurryThis(arrayBufferPrototype.slice);
+export const arrayBufferTransferToFixedLength =
+  // @ts-expect-error absent from Node 20, which we still support
+  arrayBufferPrototype.transferToFixedLength
+    ? // @ts-expect-error absent from Node 20, which we still support
+      uncurryThis(arrayBufferPrototype.transferToFixedLength)
+    : (arrayBuffer, newLength = arrayBuffer.byteLength) =>
+        // There is no `transferToFixedLength` on Node 20, which we still support.
+        // In that case, there is no way just within the language to detach an
+        // ArrayBuffer, so the best we can do is emulate it using `slice`.
+        // Unfortunately, this leaves the original non-detached.
+        arrayBufferSlice(arrayBuffer, 0, newLength);
+//
```

Comment:
Done. But made tolerant on the absence of `structuredClone` since it is not part of official JS and does not necessarily exist on all conforming platforms.